classes/class-twentytwenty-customize.php

```diff
@@ -75,8 +75,85 @@ public static function register( $wp_customize ) {
 				)
 			);
 
+			// Header & Footer Background Color.
+			$wp_customize->add_setting(
+				'header_footer_background_color',
+				array(
+					'default'           => '#ffffff',
+					'sanitize_callback' => 'sanitize_hex_color',
+					'transport'         => 'postMessage',
+				)
+			);
+
+			$wp_customize->add_control(
+				new WP_Customize_Color_Control(
+					$wp_customize,
+					'header_footer_background_color',
+					array(
+						'label'    => esc_html__( 'Header & Footer Background Color', 'twentytwenty' ),
+						'section'  => 'colors',
+						'priority' => 10,
+					)
+				)
+			);
+
 			/**
-			 * Colors.
+			 * Implementation for the accent color.
+			 * This is different to all other color options because of the accessibility enhancements.
+			 * The control is a hue-only colorpicker, and there is a separate setting that holds values
+			 * for other colors calculated based on the selected hue and various background-colors on the page.
+			 *
+			 * @since 1.0.0
+			 */
+
+			// Add the setting for the hue colorpicker.
+			$wp_customize->add_setting(
+				'accent_hue',
+				array(
+					'default'           => 344,
+					'type'              => 'theme_mod',
+					'sanitize_callback' => 'absint',
+					'transport'         => 'postMessage',
+				)
+			);
+
+			// Add setting to hold colors derived from the accent hue.
+			$wp_customize->add_setting(
+				'accent_accessible_colors',
+				array(
+					'default'           => array(
+						'content'       => array(
+							'text'   => '#000000',
+							'accent' => '#cd2653',
```

Comment:
Hmmmm and how would we calculate the secondary color?
Just 43% between background and text? I'm saying 43 specifically because `#6d6d6d` is `hsl(0,0%,43%)` so 43% textcolor - 57% background (assuming white background & black text)

Reply:
Now calculating and saving both the border & secondary colors.